README.md

```diff
@@ -217,6 +217,8 @@ Furthermore, serde imposes the following restrictions for data to roundtrip:
 - internally tagged newtype variants, which are `#[serde(flatten)]`ed together with other fields, must not contain:
   - a unit or unit struct or an untagged unit variant
 
+While RON offers a best-effort implementation for `#[serde(flatten)]`, it may be unsupported in further cases and combinations not listed above. These limitations stem primarily from Serde rather than RON. Enumerating all such cases based on Serde's behavior is nontrivial, so the lists above are not exhaustive.
```

Comment:
```suggestion
While RON offers a best-effort implementation for `#[serde(flatten)]`, it may be unsupported in further cases and combinations not listed above. These limitations stem primarily from serde rather than RON. Enumerating all such cases based on serde's behavior is nontrivial, so the lists above are not exhaustive.
```